cpu/em_smt_folding_test.py

```diff
@@ -55,9 +55,10 @@ def setUp(self):
         for package in deps:
             if not smg.check_installed(package) and not smg.install(package):
                 self.cancel("%s is needed for the test to be run" % package)
-        tarball = self.fetch_asset('http://liquidtelecom.dl.sourceforge.net'
-                                   '/project/ebizzy/ebizzy/0.3'
-                                   '/ebizzy-0.3.tar.gz')
+
+        tarball = self.fetch_asset('http://sourceforge.net/projects'
```

Comment:
Can this be made yaml param so that it can be configured as again it may fail?

Reply:
sure that is a good suggestion, will do